packages/vsce/src/commands/inspectResourceCommand.ts

```diff
@@ -9,64 +9,19 @@
  *
  */
 
-import { commands, ExtensionContext, window, workspace } from "vscode";
-import { ResourceInspectorViewProvider } from "../trees/ResourceInspectorViewProvider";
-import { getFocusRegion } from "./setFocusRegionCommand";
-import { IFocusRegion } from "./IFocusRegion";
-import { Resource, ResourceContainer } from "../resources";
-import { IResource, IResourceMeta } from "../doc";
-import { SupportedResources } from "../model";
-import { CICSLogger } from "../utils/CICSLogger";
-import { CICSMessages } from "../constants/CICS.messages";
+import { commands, ExtensionContext, TreeView, workspace } from "vscode";
 
-export function getInspectResourceCommand(context: ExtensionContext) {
-  return commands.registerCommand("cics-extension-for-zowe.inspectResource", async (resourceName: string, resourceType: string) => {
+import { inspectResourceByName } from "./inspectResourceCommandUtils";
+
+export function getInspectResourceCommand(context: ExtensionContext, treeview: TreeView<any>) {
+
+  return commands.registerCommand("cics-extension-for-zowe.inspectResource", async (resourceName: string, resourceType?: string) => {
```

Comment:
Is this command going to be used for command palette eventually as well?
If yes, both are optional.
If no, both are required.
... I think!
```suggestion
  return commands.registerCommand("cics-extension-for-zowe.inspectResource", async (resourceName: string, resourceType: string) => {
```